google-datacatalog-connectors-commons/tests/google/datacatalog_connectors/commons/utils/region_tag_helper_test.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from google.datacatalog_connectors.commons import utils
+
+
+class RegionTagHelperTestCase(unittest.TestCase):
```

Comment:
Maybe it's worth adding a test case to let clear the expected behavior when there are more than one valid region tags in the input string. Makes sense?

Reply:
I was not thinking about supporting multiple region tags for this first version, but it is a good point... Since the effort was low, I've changed a little bit to support it.